hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```diff
@@ -79,16 +79,18 @@ private ContainerUtils() {
   public static ContainerCommandResponseProto logAndReturnError(
       Logger log, StorageContainerException ex,
       ContainerCommandRequestProto request) {
-    String logInfo = "Operation: {} , Trace ID: {} , Message: {} , " +
-        "Result: {} , StorageContainerException Occurred.";
+    String logInfo = "Operation: {} , Trace ID: {} , Block ID: {} , " +
+        "Message: {} , Result: {} , StorageContainerException Occurred.";
     if (ex.getResult() == CLOSED_CONTAINER_IO ||
         ex.getResult() == CONTAINER_NOT_OPEN) {
       if (log.isDebugEnabled()) {
         log.debug(logInfo, request.getCmdType(), request.getTraceID(),
+            request.getGetBlock().getBlockID(),
             ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
       }
     } else {
-      log.info(logInfo, request.getCmdType(), request.getTraceID(),
+      log.warn(logInfo, request.getCmdType(), request.getTraceID(),
+              request.getGetBlock().getBlockID(),
```

Comment:
`request.getGetBlock()` will be `null` for most types of requests, resulting in NPE.

Reply:
> `request.getGetBlock()` is only applicable to `GetBlock` requests. Also, not all commands are related to blocks anyway.
> 
> Looking at [HDDS-8010](https://issues.apache.org/jira/browse/HDDS-8010), I think the goal is to change the message in:
> 
> https://github.com/apache/ozone/blob/4afa68e4605dd3b0c9692877a1e9ddf85ce7a740/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/BlockManagerImpl.java#L353-L364

thanks you so much
because the log 

> @GAOJHIHCYUAN have you observed the updated message in the log?

No,I haven't
I will try how could I observe the updated message.
sorry,i was a newbie. I'm still learning.
thanks you so much.

Reply:
You can update message in getBlockByID while throwing StorageContainerException to print the blockID. Can you update the PR?

Reply:
> You can update message in getBlockByID while throwing StorageContainerException to print the blockID. Can you update the PR?

okay,I will update the PR.
appreciate